docs/design/api/storage.rst

```diff
@@ -0,0 +1,324 @@
+Storage API
+###########
+
+For this examples let's say:
+
+- Buyers logs with Firefox Account Bearer Tokens
+- Seller apps logs with Hawk
+- Payment App logs with Basic Auth
+
+
+As a user, I want to retrieve my information about my purchases
+===============================================================
+
+.. code-block:: http
+
+    GET /buckets/mozilla/collections/payments/records
```

Comment:
It is because we are building a generic storage service so we are using collection and records.

Also a specific record will be:

    GET /buckets/mozilla/collections/payments/records/<record_id>